sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightReporter.cs

```diff
@@ -140,6 +154,7 @@ private void InitializePlaywrightReporter(string xmlSettings)
             EnableGithubSummary = _enableGitHubSummary,
             TestRunStartTime = DateTime.UtcNow,
             AccessTokenDetails = tokenDetails,
+            NumberOfTestWorkers = numberOfTestWorkers != null ? Convert.ToInt32(numberOfTestWorkers) : 1
```

Comment:
When user is not passing it, i assume we would make 1 as default, Can we make it NA or some value which means we dont know, because As per Nunit docs, default value it uses is depend on Number of Core of machine So 1 as default seems wrong.

Reply:
Reporting service supports int only